pydatastructs/utils/misc_util.py

```diff
@@ -297,3 +298,26 @@ def __new__(cls, key, data):
         obj.data = data
         obj.parent, obj.size = [None]*2
         return obj
+
+'''class BTreeNode(object):
+
+    __slots__ = ['keys', 'degree', 'children', 'data', 'is_root', 'no_of_keys', 'search', 'traverse']
+
+    def __new__(cls, key, data, degree, children):
+        obj = Node.__new__(cls)
+        obj.data, obj.key = data, key
+        obj.degree = degree
+        obj.no_of_keys = no_of_keys
+        obj.children = [] or children
+        obj.is_root = False
+        return obj
```

Comment:
https://github.com/codezonediitj/pydatastructs/blob/692624605aeb8a47c33f82e4838f8a01b8af5d3d/pydatastructs/utils/misc_util.py#L136-L143

No of keys can be figured out from no of children.
Is degree the same as order in Knuth's definition.

Reply:
Degree represents the lower bound on the number of children a node in the B Tree can have (except for the root). i.e the minimum number of children possible. Whereas the Order represents the upper bound on the number of children. ie. the maximum number possible.

Reply:
> Every non-leaf node (except root) has at least ⌈m/2⌉ child nodes.

Can degree be computed from order?